src/turnkeyml/build/export.py

```diff
@@ -279,9 +280,31 @@ def fire(self, state: build.State):
         default_warnings = warnings.showwarning
         warnings.showwarning = _warn_to_stdout
 
+        stats = fs.Stats(
+            state.cache_dir, state.config.build_name, state.evaluation_id
+        )
+
+        # Verify if the exported model matches the input torch model
+        try:
+            # The `torch.onnx.verification.find_mismatch()` takes input arguments to the
+            # model as `input_args (Tuple[Any, ...])`
+            export_verification = torch.onnx.verification.find_mismatch(
+                state.model,
+                tuple(state.inputs.values()),
+                opset_version=state.config.onnx_opset)
```

Comment:
How does this actually work? You aren't passing it the ONNX file, so what is it checking against?

Reply:
This works by comparing the exported model with OnnxRuntime
Source reference [here](https://github.com/pytorch/pytorch/blob/1dd4813328a0f436834b7493afcac6f0f319a4bd/torch/onnx/verification.py#L1648)